pkg/logs/query.go

```diff
@@ -72,36 +72,36 @@ func Query(
 
 	req, err := http.NewRequest(http.MethodGet, endpoint.String(), nil)
 	if err != nil {
-		return warn, errors.Wrap(err, "creating request")
+		return http.StatusBadRequest, warn, errors.Wrap(err, "creating request")
```

Comment:
How do we know this is bad request? We haven't reached the server yet.

Reply:
👍 resorted to `0` as the status code.